SU2_CFD/src/python_wrapper_structure.cpp

```diff
@@ -445,34 +366,24 @@ passivedouble CDriver::GetVertexForceDensityZ(unsigned short iMarker, unsigned l
 
 void CDriver::SetVertexCoordX(unsigned short iMarker, unsigned long iVertex, passivedouble newPosX) {
 
-  unsigned long iPoint;
-  su2double *Coord;
-
-  iPoint = geometry_container[ZONE_0][INST_0][MESH_0]->vertex[iMarker][iVertex]->GetNode();
-  Coord = geometry_container[ZONE_0][INST_0][MESH_0]->nodes->GetCoord(iPoint);
+  auto iPoint = geometry_container[ZONE_0][INST_0][MESH_0]->vertex[iMarker][iVertex]->GetNode();
+  auto Coord = geometry_container[ZONE_0][INST_0][MESH_0]->nodes->GetCoord(iPoint);
 
   PyWrapVarCoord[0] = newPosX - Coord[0];
-
 }
 
 void CDriver::SetVertexCoordY(unsigned short iMarker, unsigned long iVertex, passivedouble newPosY) {
 
-  unsigned long iPoint;
-  su2double *Coord;
-
-  iPoint = geometry_container[ZONE_0][INST_0][MESH_0]->vertex[iMarker][iVertex]->GetNode();
-  Coord = geometry_container[ZONE_0][INST_0][MESH_0]->nodes->GetCoord(iPoint);
+  auto iPoint = geometry_container[ZONE_0][INST_0][MESH_0]->vertex[iMarker][iVertex]->GetNode();
```

Comment:
These functions also were all intended to be used with the old mesh solver. Now that we do not provide anymore the change in coordinates to the geometry container but to the mesh solver they should not be used anymore.

Reply:
If it does not work it should be removed.

Reply:
okok, I will do that